src/app/app-routing.module.ts

```diff
@@ -1,7 +1,8 @@
 import { NgModule } from '@angular/core';
 import { RouterModule, Routes } from '@angular/router';
 
-import { IsNotLoggedInGuard } from './auth/guards/is-not-logged-in.guard';
+import { IsLoggedInGuard } from "@guards/is-logged-in.guard";
```

Comment:
It's not exactly what did I mean. Module contains a lot of imports. You shouldn't be confused about them. Both guards refer to authentication.
Make it like this:
`@auth/guards`

In this case you will receive:
`import { IsLoggedInGuard, IsNotLoggedInGuard } from "@auth/guards";`